src/systems/systems/src/hydraulic/landing_gear.rs

```diff
@@ -353,6 +353,18 @@ enum GearSysComponentId {
     Door,
     Gear,
 }
+impl GearSysComponentId {
+    fn from_actuator_id(id: GearActuatorId) -> Self {
```

Comment:
I think in Rust you typically implement the `From` trait for an operation like this.

Reply:
Indeed :)